#!/usr/bin/env python
PACKAGE = "franka_example_controllers"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()
#          name                   paramtype level desciption                    default=? min  max
gen.add("task1_Kp", double_t, 0, "task1_Kp", 50, 0, 400)#
gen.add("task1_Kv", double_t, 0, "task1_Kv", 20, 0, 30)#

gen.add("task2_Md", double_t, 0, "inertia", 3, 0, 100)#
gen.add("task2_Bd", double_t, 0, "damping", 8, 0, 100)
gen.add("task2_Kd", double_t, 0, "stiffness", 4, 0, 100)
#pkgname nodename name
exit(gen.generate(PACKAGE, "dynamic_compliance", "null_space_controller_param"))
